extras/ortoa/benchmark/infrastucture/stats_collection.py

```diff
@@ -44,13 +40,25 @@ def _graph_byte_size(self, dir: Path) -> None:
         )
         fig = ax.get_figure()
         fig.savefig(dir / "byte_size.pdf")
+    
+    def _graph_db_size(self, dir: Path) -> None:
+        ax = self.raw_df.plot.bar(
+            x="db_size",
+            y=["average_latency", "throughput"],
+            secondary_y=["average_latency"],
+        )
+        fig = ax.get_figure()
+        fig.savefig(dir / "db_size.pdf")
```

Comment:
I didn't actually run the benchmark to test this due to not having the updated concurrency changes but I am quite confident it should work.